lib/charms/layer/jenkins/plugins.py

```diff
@@ -68,17 +69,15 @@ def install(self, plugins):
         @params plugins: A whitespace-separated list of plugins to install.
         """
         hookenv.log("Starting plugins installation process")
-        plugins = plugins or ""
-        plugins = plugins.split()
+        plugins = itertools.chain(REQUIRED_PLUGINS, (plugins or "").split())
         plugins, incompatible_plugins = self._get_plugins_to_install(plugins)
         if len(incompatible_plugins) != 0:
-            hookenv.log("The following plugins require a higher jenkins version"
-                        " and were not installed: (%s)" % " ".join(
-                            incompatible_plugins))
-        configured_plugins = self._get_plugins_to_install(
-            hookenv.config()["plugins"].split())
-        host.mkdir(
-            paths.PLUGINS, owner="jenkins", group="jenkins", perms=0o0755)
+            hookenv.log(
+                "The following plugins require a higher jenkins version"
+                " and were not installed: (%s)" % " ".join(incompatible_plugins)
+            )
```

Comment:
Bump the log level to WARNING or ERROR? The default log level is INFO which seems a little bit too low for this situation.

Reply:
Good suggestion, out of scope for this PR since this was only modified due to formatting, I should have disabled auto formatting